parser_test.go

```diff
@@ -709,8 +709,10 @@ AAAAAAAAAAAA5gm5Mg==
 
 		// for show table regions.
 		{"show table t1 regions", true, "SHOW TABLE `t1` REGIONS"},
+		{"show table t1 regions where a=1", true, "SHOW TABLE `t1` REGIONS WHERE `a`=1"},
 		{"show table t1", false, ""},
 		{"show table t1 index idx1 regions", true, "SHOW TABLE `t1` INDEX `idx1` REGIONS"},
+		{"show table t1 index idx1 regions where a=2", true, "SHOW TABLE `t1` INDEX `idx1` REGIONS WHERE `a`=2"},
```

Comment:
Also add two test cases for `show table t1 regions like 'something%'`?

Reply:
done.

Reply:
@crazycs520 I mean literal ``SHOW TABLE `t1` INDEX `idx1` REGIONS LIKE 'some%'``, not ``WHERE `a` LIKE 'some%'``. Don't use `ShowLikeOrWhereOpt` if you can't support that...

Reply:
Great, thanks. PTAL